src/main/resources/templates/db_scripts/schema.json

```diff
@@ -462,6 +456,11 @@
         }
       ],
       "fullTextIndex": [
+        {
+          "fieldName": "classifications",
+          "arraySubfield" : "classificationNumber",
+          "arrayModifiers": ["classificationTypeId"]
```

Comment:
Why have these been added? Are they necessary for using `classifications` as a full text index with the same CQL as before?

Reply:
I modeled this on how other full-text indexes in the same table that index arraylist fields are structured.  Contributors, for example, is structured much like classifications, as a list which contains multiple subfields.  In that case, the name is what people actually want to search.  In this case, it's the classification number.  I'm assuming you specify a subfield because indexing all subfields could lead to false matches-for example, a search on classification number which also happens to match part of the classification ID.  Arraymodifiers I'm less sure about-my assumption there is that it possibly allows you to specify searching for a classification number under a specific classification type.  Since other similar indexes have that capability, I included it here as well.  I don't think it's required for search per se.

Reply:
Ok, this seems speculative to me, given that folks aren't using CQL with those modifiers and haven't asked to use them.

With the aim of expediency, unless @julianladisch suggests we shouldn't add them, I'm going to suggest we leave them in

Reply:
The schema.json change is perfect.

Reply:
> The schema.json change is perfect.

Thanks @julianladisch